internal/pkg/plugin/trello/update.go

```diff
@@ -0,0 +1,28 @@
+package trello
+
+import (
+	"github.com/merico-dev/stream/pkg/util/log"
+)
+
+// Update remove and set up trello workflows.
+func Update(options map[string]interface{}) (map[string]interface{}, error) {
+
+	var opt *Options
+	var err error
+
+	if opt, err = convertMap2Options(options); err != nil {
+		return nil, err
+	}
+
+	if err := validateOptions(opt); err != nil {
+		return nil, err
+	}
+
+	trelloIds, err := CreateTrelloItems(opt)
+	if err != nil {
+		return nil, err
+	}
+	log.Success("Creating trello board succeeded.")
```

Comment:
I think in the core, we have logging for the success of each plugin, right? So, this might be redundant information? Please run some tests to make sure the logging output makes sense to a user (i.e., order, no redundant info, etc.)